wire-ios-system/Source/Logging/WireLogger.swift

```diff
@@ -34,46 +40,46 @@ public struct WireLogger: LoggerProtocol {
     // MARK: - LoggerProtocol
 
     public var logFiles: [URL] {
-        Self.provider.logFiles
+        Self.provider?.logFiles ?? []
```

Comment:
I made `provider` optional, because for unit tests the `initialize` method is not called.
Do you agree with this approach or do you suggest something different?